src/main/resources/graphql/schema/user.graphqls

```diff
@@ -14,10 +14,6 @@ input UserLoginDTO {
     password: String !
 }
 
-extend type Mutation {
-    createUser(user: UserDTO !): User !
```

Comment:
What do you think about actually inverted approach? That means getting rid of `createVenue` and `createBand` mutations. Instead create `createUserBand(user: UserDTO !, band: BandDTO !)` and `createUserVenue(user: UserDTO !, venue: VenueDTO !)` mutations - in that case we can even consider removing `role` from `UserDTO`.

Reply:
hmm splitting this like this maybe make sense

Reply:
removing role would get us out of String Driven Development